### PR TITLE
Deprecate Old Build Files

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Removed all occurrences of instance_ips.txt, instance_ids.txt and, images_data.txt and replaced with env_result.json file when relevant.
- Removed the following files:
    lastest_server_build_scripts/create_instance.sh
    create_instance.sh,
    create_instances.py,
    get_instance_ip.sh
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Tests/scripts/run_content_installation.py [hunk 1]
```diff
@@ -6,27 +6,10 @@
...
-        with open('./Tests/images_data.txt', 'a') as instance_file:
-            instance_file.write('{} Image info is: {} {} {}\n'.format(env["Role"], env["AmiId"], env["AmiName"],
-                                                                      env["AmiCreation"]))
-
-    with open('./Tests/instance_ids.txt', 'w') as instance_file:
-        instance_file.write('\n'.join(instance_ids))
-
-    with open('instance_ids', 'w') as instance_file:
-        instance_file.write('\n'.join(instance_ids_nonami))
+    id_to_ip = {env["InstanceID"]: env["InstanceDNS"] for env in env_results}
```
Comment (yaakovi) [inline Tests/scripts/run_content_installation.py:12]: don't see a reason to keep it as a dict, looks like we only use the values.
convert to a list.

File: Tests/test_content.py [hunk 2]
```diff
@@ -716,6 +717,14 @@ def run_test_scenario(tests_queue, tests_settings, t, proxy, default_test_timeou
              is_ami, thread_index=thread_index)
 
 
+def load_env_results_json():
+    if not os.path.isfile(ENV_RESULTS_PATH):
+        return None
```
Comment (yaakovi) [inline Tests/test_content.py:722]: return an empty dict so the type of all return values will be the same.

**Review by yaakovi — APPROVED** [review]

**Comment by yaakovi** [comment]
before merging, please run a "non-AMI" flow to make sure nothing breaks.
trigger script:
`Utils/trigger_content_non_ami_build.sh`

**Comment by Bargenish** [comment]
@yaakovi 
Non-AMI flow:
Screen Shot 2020-10-11 at 10 23 54

**Comment by yaakovi** [comment]
looks good, you may merge the PR.

**Pull request merged**